theme6/source/partials/_header.erb

```diff
@@ -0,0 +1,102 @@
+
+	<!-- version mobile -->
+<script src="../javascripts/site.js"></script>
+ 	<div class="navbar">
+ 		<div class="menubar">
+  <!-- SIDEBAR -->
+ 			<div id="mySidenav" class="sidenav">
+ 				<div class="sidenav_logo">
+ 					<img class="sidenavhorse" src="../images/logo2.png">
+ 					<a href="javascript:void(0)" class="closebtn" onclick="closeNav()"><p>Close <img class="arrow_back" src="../images/Path 322 Copy 3@2x.png"></p></a>
```

Comment:
Let's make this as a button, add a class or an ID on it and use that using JS to call the function `closeNav()`;